src/test/java/ca/concordia/soen6441/project/gameplay/orders/AirliftTest.java

```diff
@@ -63,4 +63,15 @@ void testAirliftExecution() {
         verify(d_mockSourceCountry).setTroops(5);   // 10 - 5
         verify(d_mockTargetCountry).setTroops(15);  // 10 + 5
     }
+    @Test
+    void testValidate_ReturnsError_WhenPlayerDoesNotOwnSourceCountry() {
```

Comment:
No, we need a test for this in IssueOrder.  This is when the order is issued (and not when its executed).
Please create a test as such, and resubmit again.